src/layouts/MainLayout/MainLayout.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useNavigate, useOutlet } from 'react-router-dom';
+import { paths } from '../../shared';
+
+export function MainLayout() {
```

Comment:
I don't really like the name `Main` layout. Maybe dashboard or panel? There could be different layouts and main does not specifically explain the contents of itself